complement/pipeline.yml

```diff
@@ -0,0 +1,38 @@
+steps:
+  - command:
+      - "env"
+      - "go build -tags *"
```

Comment:
uhh what does this do?

Reply:
`go build ./internal/...` would be helpful to make sure there's no weird typos in Complement such that it doesn't compile.

Reply:
Well, actually the `*` needs to be wrapped in quotes. But it builds all optional build tags.

Is there a reason we wouldn't want to build a specific tag in CI?

Reply:
Nope, we should build everything we can.